packages/flutter/lib/src/widgets/sheet.dart

```diff
@@ -0,0 +1,677 @@
+import 'dart:async';
+import 'dart:ui';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+
+const Curve _decelerateEasing = Cubic(0.0, 0.0, 0.2, 1.0);
+const Curve _modalBottomSheetCurve = _decelerateEasing;
+const Duration _bottomSheetDuration = Duration(milliseconds: 400);
+const double _minFlingVelocity = 500.0;
+const double _closeProgressThreshold = 0.6;
+const double _willPopThreshold = 0.8;
+
+// Arbitrary value
+const Duration _kBounceAnimationDuration = Duration(milliseconds: 300);
+
+/// A custom bottom sheet.
+///
+/// The [Sheet] widget itself is rarely used directly. Instead, prefer to
+/// create a modal bottom sheet with [showMaterialModalBottomSheet].
+///
+/// See also:
+///
+///  * [showMaterialModalBottomSheet] which can be used to display a modal bottom
+///    sheet with Material appareance.
+///  * [showCupertinoModalBottomSheet] which can be used to display a modal bottom
```

Comment:
I can't find these anywhere. How do I use the sheet API as a developer?

Reply:
This is removed now, it was part of the old API I use in the original package 
https://github.com/jamesblasco/modal_bottom_sheet